lib/web_ui/lib/src/ui/test_embedding.dart

```diff
@@ -29,17 +29,21 @@ Future<dynamic> ensureTestPlatformInitializedThenRunTest(
 /// are available.
 Future<void> _platformInitializedFuture;
 
-/// Initializes domRenderer with specific devicePixelRation and physicalSize.
+/// Initializes domRenderer with specific devicePixelRatio and physicalSize.
 Future<void> webOnlyInitializeTestDomRenderer({double devicePixelRatio = 3.0}) {
   // Force-initialize DomRenderer so it doesn't overwrite test pixel ratio.
   engine.domRenderer;
 
-  // The following parameters are hard-coded in Flutter's test embedder. Since
-  // we don't have an embedder yet this is the lowest-most layer we can put
-  // this stuff in.
-  engine.window.debugOverrideDevicePixelRatio(devicePixelRatio);
-  engine.window.webOnlyDebugPhysicalSizeOverride =
-      Size(800 * devicePixelRatio, 600 * devicePixelRatio);
+  assert(() {
```

Comment:
Why is this assert necessary? This code emulates the `flutter test` environment (it sets `debugEmulateFlutterTesterEnvironment` to true below). So it shouldn't affect e2e tests. Those should not hit this code.

Reply:
reverted